### DISCUSSION TITLE
Finish print notification

### DISCUSSION THREAD

**Discussion by Shadow-FW** [nlef/moonraker-telegram-bot#214] [Q&A] [answered]
Hello. Is it possible to make the bot not edit the message at the end of printing, but send a new one? I need to hear a notification about the end of printing more than about the beginning.

**Comment by aka13-404** [comment, marked as answer]
No, but you can send a notification whenever you like with RESPOND- messages.
Just add it to your endcode, and you will know that your print is finished.

**Reply by Shadow-FW** [reply]
Thanks. On one host is working good, but on another one have error: {'message': "'ascii' codec can't decode byte 0xd0 in position 8: ordinal not in range(128)", 'error': 'WebRequestError'}
